File: auth-service/src/main/scala/com/hashmap/haf/AuthApplication.scala
package com.hashmap.haf

import org.springframework.boot.{SpringApplication, SpringBootConfiguration}
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.context.annotation.ComponentScan

@EnableDiscoveryClient
@SpringBootConfiguration
@EnableAutoConfiguration
@ComponentScan
class AuthApplication

object AuthApplication extends App{
	SpringApplication.run(classOf[AuthApplication], args: _*)
}

File: auth-service/src/main/java/com/hashmap/haf/AuthApplication.java
package com.hashmap.haf;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.SpringBootConfiguration;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.context.annotation.ComponentScan;

@EnableDiscoveryClient
@SpringBootConfiguration
@EnableAutoConfiguration
@ComponentScan
public class AuthApplication {
    public static void main(String[] args) {
        SpringApplication.run(AuthApplication.class, args);
    }
}
